src/app/components/community/sidebar/sidebar.ts

```diff
@@ -79,35 +61,37 @@ export default class AppCommunitySidebar extends Vue {
 	}
 
 	get shouldShowKnownMembers() {
-		return !!this.app.user && this.knownMembers && this.knownMembers.length > 0;
+		return !!this.app.user && this.data.knownMembers && this.data.knownMembers.length > 0;
 	}
 
 	get membersYouKnowCount() {
-		return number(this.knownMemberCount);
+		return number(this.data.knownMemberCount);
 	}
 
 	get shareUrl() {
-		return Environment.baseUrl + this.$router.resolve(this.community.routeLocation).href;
+		return Environment.baseUrl + this.$router.resolve(this.data.community.routeLocation).href;
```

Comment:
Check reactivity. now that we're passing a community through a data object from a payload and not directly from a prop we have two separate instances of the community to react on.

Maybe we should keep taking the community as a prop since the community is the "subject" the sidebar is attached to?